packages/region-config-resolver/README.md

```diff
@@ -0,0 +1,12 @@
+# @aws-sdk/region-config-resolver
```

Comment:
should say smithy

also, the commit message should be `feat(region-config-provider): xxx` or you can leave that out because smithy-ts hasn't traditionally used that prefix system

Reply:
Isn't resolving regions AWS-specific? I think it should still be in the `@aws-sdk` scope